### PR TITLE
Started working for scenario tests for CuberitePluginChecker

### PR DESCRIPTION
**Pull request by NiLSPACE**
With this PR WorldEdit works without any problems in the CuberitePluginChecker.

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -41,13 +42,13 @@ for _, Folder in ipairs(cFile:GetFolderContents(WorldEditPath)) do repeat
...
 
-PLUGIN = nil
+
+
 
 
 function Initialize(a_Plugin)
-	PLUGIN = a_Plugin
-	PLUGIN:SetName(g_PluginInfo.Name)
-	PLUGIN:SetVersion(g_PluginInfo.Version)
```
Comment (madmaxoft) [inline main.lua]: I suppose we should make an exception in the Checker to allow this one specific case.

Comment (NiLSPACE) [inline main.lua]: Yeah, probably.

Comment (madmaxoft) [inline main.lua]: Exception added. But yes, this kind of code is ugly, especially all-caps variable names.

Comment (NiLSPACE) [inline main.lua]: Should cWorld also be an exception? Technically they can be stored right? Since we do have the cRoot:GetWorld method.

**Comment by NiLSPACE** [comment]
Do you mind if I merge this?

**Comment by madmaxoft** [comment]
Sure, go ahead, you're the main WE man, after all :)

**Comment by madmaxoft** [comment]
Although, I think you should first connect the CIs, test this PR and only merge after success.

**Pull request merged**